### ISSUE TITLE
plan _system_ not found on v1.3.0

### ISSUE THREAD

**Issue by http-kennedy** [garethgeorge/backrest#383]
When running "Check Now" or "Prune" getting error message that _system_ not found.

**To Reproduce**
Repo Settings -> Prune Now or Check Now

**Screenshots**
![image](https://github.com/user-attachments/assets/edf0ccdf-e8e0-4a0a-8286-77152edc9d5b)
![image](https://github.com/user-attachments/assets/4459827f-cb8e-447a-af19-b8d6563241c5)

**Platform Info**
- Docker Container
- Backrest Version 1.3.0

**Additional context**
compose.yaml

```
version: "3.2"
services:
  backrest:
    image: garethgeorge/backrest
    container_name: backrest
    hostname: backrest
    volumes:
      - /root/.config/backrest/data:/data
      - /root/.config/backrest/config:/config
      - /root/.config/backrest/cache:/cache
    environment:
      - BACKREST_DATA=/data
      - BACKREST_CONFIG=/config/config.json
      - XDG_CACHE_HOME=/cache
    restart: unless-stopped
    ports:
      - 9898:9898
networks: {}
```

**Comment by garethgeorge** [comment]
Thanks for this report -- oversight in my testing. Putting out a patch this afternoon and will leave this open to track adding test coverage for the system plans over the weekend.

**Comment by http-kennedy** [comment]
> Thanks for this report -- oversight in my testing. Putting out a patch this afternoon and will leave this open to track adding test coverage for the system plans over the weekend.
> 
> They're apparently a hole in my manual testing -- I don't think I noticed my hooks failing when testing forget / prune.

Thank you for the quick reply! Appreciate your work.

**Comment by garethgeorge** [comment]
Fixed in 1.3.1, thanks again for the quick report
